plugins/GSdx/GSRendererHW.cpp

```diff
@@ -201,22 +201,23 @@ void GSRendererHW::InvalidateLocalMem(const GIFRegBITBLTBUF& BITBLTBUF, const GS
 	m_tc->InvalidateLocalMem(m_mem.GetOffset(BITBLTBUF.SBP, BITBLTBUF.SBW, BITBLTBUF.SPSM), r);
 }
 
-int GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
+uint16 GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
 {
 	float t = (1.0f - alpha) * t0 + alpha * t1;
-	return (int)t & ~0xF; // cheap rounding
+	return (uint16)t & ~0xF; // cheap rounding
 }
```

Comment:
Since you're casting ``t`` immediately to ``uint16``, why not change ``t`` itself to a ``uint16`` type variable ? (this way you avoid the cast)